.github/workflows/command-fmt.yml

```diff
@@ -0,0 +1,52 @@
+name: Command-FMT
+
+on: 
+  workflow_dispatch: 
+    inputs:
+      pr:
+        description: Number of the Pull Request
+        required: true
+
+jobs:
+  cmd-fmt:
+    runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-unified:bullseye-1.75.0-2024-01-22-v20240109
```

Comment:
Sorry that I write in the middle of the developing process but it'd be easier to read container name from `.github/env` file like it's done [here](https://github.com/paritytech/polkadot-sdk/blob/master/.github/workflows/tests-linux-stable.yml#L29-L45). In this case we don't have to change the image in all action files.

Reply:
That's a great suggestion! Thanks! I will apply that